copybook4java-codegen-maven-plugin/src/main/java/com/nordea/oss/copybook/codegen/CopyBookConverter.java

```diff
@@ -29,24 +29,25 @@ public CopyBookConverter() throws Exception {
         InputStream inputStream = this.getClass().getResourceAsStream("classconverter.html");
         String js = extractJS(inputStream);
 
-        manager = new ScriptEngineManager(null);
+        manager = new ScriptEngineManager();
         engine = manager.getEngineByName("nashorn");
-
-        if(engine == null) {
-            throw new Exception("Could not load the Nashorn script engine");
-        }
-
         invocable = (Invocable) engine;
-        engine.eval(js);
+        try {
+            engine.eval(js);
+        }catch (Exception e){
+            // In Java 11 nashorn throws null pointer exception.
```

Comment:
Would be nice to know if it's something in the JS code that makes it throw so maybe the whole script does not get evaluated or at least add a link if this is a bug in Nashorn?

Reply:
Yes indeed and at minimum log the exception.